src/Native/SymSgdNative/CMakeLists.txt

```diff
@@ -4,16 +4,11 @@ set(SOURCES
     SymSgdNative.cpp
 )
 
-if(WIN32)
-    find_library(MKL_LIBRARY MklImports HINTS ${CMAKE_SOURCE_DIR}/../../packages/mlnetmkldeps/0.0.0.5/runtimes/win-x64/native)
-else()
+file(TO_NATIVE_PATH "${MKL_LIB_PATH}" MKL_LIB_PATH)
```

Comment:
What does this line do?

Reply:
converts to right slash depending on the OS

---
In reply to: [208044368](https://github.com/dotnet/machinelearning/pull/647#discussion_r208044368) [](ancestors = 208044368)